arex-schedule-web-api/src/main/java/com/arextest/schedule/comparer/impl/DefaultReplayResultComparer.java

```diff
@@ -79,8 +86,17 @@ public boolean compare(ReplayActionCaseItem caseItem, boolean useReplayId) {
                 MockCategoryType.Q_MESSAGE_CONSUMER.getName().equalsIgnoreCase(caseItem.getCaseType())) {
                 return comparisonOutputWriter.writeQmqCompareResult(caseItem);
             }
-
-            caseItemRepository.updateCompareStatus(caseItem.getId(), CompareProcessStatusType.PASS.getValue());
+            CompareProcessStatusType compareStatus = CompareProcessStatusType.PASS;
```

Comment:
Is the status of the above qmq result not updated

Reply:
Qmq doesn't need to update. Cause it does't update ReplayActionCaseItemRepository.

Reply:
Why qmq doesn't need to update?